### DISCUSSION TITLE
Volumetric Extrusion

### DISCUSSION THREAD

**Discussion by novabansal** [mriscoc/Ender3V2S1#849] [Q&A]
Hi, 3D printing noob here. I'm running an Ender 3 Max Neo with Mriscoc. I'm trying to tune my e-steps and they're really high (around 700...), and someone suggested a solution might be to disable volumetric extrusion. Is that a feature in Mriscoc and where can I turn it off!

**Comment by dfinberg** [comment]
700? That's not reasonable, there's something wrong with your setup. Maybe a bad stepper connection at either end of the cable, or .. no idea. No tension on the extruder gear?

To tune the esteps you removed the bowden tube, clipped the end of the filament where it exited the extruder, and used the spiffy new extrude 100mm menu option right?

**Comment by novabansal** [comment]
I solved the issue! Just wanted to make sure it wasn't this one thing. Thank you

**Reply by AsherNoble** [reply]
How did you solve it? Im currently running mriscoc and my e-steps are calibrated at 400ish. Did you work out how to disable volumetric extrusion?

**Reply by mriscoc** [reply]
About of 400 for a Sprite extruder (or similar) seems correct.
